Backend-API/src/main/java/com/avangers/backendapi/controllers/UserController.java

```diff
@@ -5,11 +5,12 @@
 import com.avangers.backendapi.services.UserServiceImpl;
 import lombok.RequiredArgsConstructor;
 import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RestController;
 
-
+@CrossOrigin ("http://localhost:5173")
```

Comment:
**Later** we need to make this work with environment variables, since the url of the frontend will change.
Once it will be localhost, but other times it will be the railway url.